docs/learn/digital-assets/migrate-erc20-to-lsp7.md

```diff
@@ -0,0 +1,194 @@
+---
+sidebar_label: '💰 Migrate ERC20 to LSP7'
+sidebar_position: 3
+description: Learn how to migrate your ERC20 token to the LSP7 Digital Asset standard on LUKSO.
+---
+
+# 🪙 Migrate ERC20 to LSP7
+
+[LSP7DigitalAsset](../../standards/tokens/LSP7-Digital-Asset.md) is a new token standard that offers a wider range of functionality compared to [ERC20](https://eips.ethereum.org/EIPS/eip-20), as described in the [standard section](../../standards/tokens/LSP7-Digital-Asset.md). For migrating from ERC20 to LSP7, developers need to be aware of several key differences.
+
+## Smart Contract Building
+
+Usually, to create an ERC20 token, [@openzeppelin/contracts](https://www.npmjs.com/package/@openzeppelin/contracts) package will be used, and ERC20 will be inherited.
+
+```solidity title="ERC20 Token"
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract MyERC20Token is ERC20 {
+    constructor(string memory name, string memory symbol) ERC20(name, symbol) {
+        // Your constructor logic
+    }
+}
+```
+
+For LSP7, [@lukso/lsp7-contracts](https://www.npmjs.com/package/@lukso/lsp7-contracts) package will be used, and LSP7 will be inherited.
+
+```solidity title="LSP7 Token"
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.15;
+
+import "@lukso/lsp7-contracts/contracts/LSP7DigitalAsset.sol";
+
+contract MyLSP7Token is LSP7DigitalAsset {
+    constructor(
+        string memory name, // Name of the token
+        string memory symbol, // Symbol of the token
+        address tokenOwner, // Owner able to add extensions and change metadata
+        uint256 lsp4tokenType, // 0 if representing a fungible token, 1 if representing an NFT
+        bool isNonDivisible // false for decimals equal to 18, true for decimals equal to 0
+    ) LSP7DigitalAsset(name, symbol, tokenOwner, lsp4tokenType, isNonDivisible) {
+        // _mint(to, amount, force, data)
+        // force: should be set to true to allow EOA to receive tokens
+        // data: only relevant if the `to` is a smart contract supporting LSP1.
+        _mint(tokenOwner, 200000, true, "");
+    }
+}
+```
+
+## Behavior
+
+- For LSP7, **mint and transfer functions will have a `force` additional field**. For full compatibility with ERC20 behavior (where the recipient can be any address), set this to `true`. Setting it to `false` will only allow the transfer to smart contract addresses supporting the LSP1 interfaceId. (Check [LSP1UniversalReceiver section](../../standards/tokens/LSP7-Digital-Asset.md#lsp1-token-hooks) in LSP7DigitalAsset for more info).
```

Comment:
Maybe it would be useful to write next to each other the `transfer` function at least for ERC20 and 721?